File: order-producer/src/main/java/com/sahakav/coffee_shop/repository/CoffeeOrderRepository.java
package com.sahakav.coffee_shop.repository;

import com.sahakav.coffee_shop.model.CoffeeOrder;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.UUID;

@Repository
public interface CoffeeOrderRepository extends JpaRepository<CoffeeOrder,UUID> {
}

File: order-producer/src/main/scala/com/sahakav/coffee_shop/repository/CoffeeOrderRepository.scala
package com.sahakav.coffee_shop.repository

import java.util.UUID

import com.sahakav.coffee_shop.model.CoffeeOrder
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
trait CoffeeOrderRepository extends JpaRepository[CoffeeOrder, UUID]{

}
